src/api/schema.rs

```diff
@@ -0,0 +1,45 @@
+use async_graphql::validators::IntRange;
+use async_graphql::{
+    EmptyMutation, FieldResult, Object, Schema, SchemaBuilder, SimpleObject, Subscription,
+};
+use chrono::{DateTime, Utc};
+use tokio::stream::{Stream, StreamExt};
+use tokio::time::Duration;
+
+#[SimpleObject]
+struct Heartbeat {
+    utc: DateTime<Utc>,
+}
+
+impl Heartbeat {
+    fn new() -> Self {
+        Heartbeat { utc: Utc::now() }
+    }
+}
+
+pub struct Query;
+
+#[Object]
+impl Query {
+    /// Returns `true` to denote the GraphQL server is reachable
+    async fn health(&self) -> FieldResult<bool> {
+        Ok(true)
+    }
+}
+
+pub struct Subscription;
+
+#[Subscription]
+impl Subscription {
+    async fn heartbeat(
```

Comment:
Is there some way to convert Rust doc comments into GraphQL API documentation?

Would be nice to explicitly mention that the interval is defined in milliseconds (and extend that documentation to any future subscriptions/queries/mutations).

Reply:
In Juniper, `///` doc comments transform to GraphQL schema documentation. Haven't tried that in async-graphql, actually -- that'll be interesting to see.

Reply:
Nice, `///` -> schema docs:

<img width="632" alt="Screenshot 2020-08-24 at 13 56 18" src="https://user-images.githubusercontent.com/719722/91047331-c6a8b880-e611-11ea-808a-aa21b37f2f2c.png">